From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: ZHANG Yuntian <yt@radxa.com>
Date: Wed, 26 Feb 2025 09:18:00 +0000
Subject: [PATCH] fix: disable Allwinner cross toolchain

Signed-off-by: ZHANG Yuntian <yt@radxa.com>
---
 src/Makefile | 44 --------------------------------------------
 1 file changed, 44 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index eca6c91aab7..4755175baa8 100755
--- a/src/Makefile
+++ b/src/Makefile
@@ -271,50 +271,6 @@ ifeq (x$(config_check), xyes)
 endif
 endif
 
-#########################################################################
-ifeq (x$(CONFIG_CPU_E90X), xy)
-RISCV_PATH=Xuantie-900-gcc-linux-5.10.4-glibc-x86_64-V2.8.1
-else ifeq (x$(CONFIG_ARCH_RV32I), xy)
-RISCV_PATH=nds32le-linux-glibc-v5d
-else
-RISCV_PATH=riscv64-linux-x86_64-20200528
-endif
-
-riscv_toolchain_check=$(shell if [ ! -d ../tools/toolchain/$(RISCV_PATH) ]; then echo yes; else echo no; fi;)
-ifeq (x$(riscv_toolchain_check), xyes)
-$(info Prepare riscv toolchain ...);
-$(shell mkdir -p ../tools/toolchain/$(RISCV_PATH) || exit 1)
-$(shell tar --strip-components=1 -xf ../tools/toolchain/$(RISCV_PATH).tar.xz -C ../tools/toolchain/$(RISCV_PATH) || exit 1)
-endif
-arm_toolchain_check=$(shell if [ ! -d ../tools/toolchain/gcc-linaro-7.2.1-2017.11-x86_64_arm-linux-gnueabi ]; then echo yes; else echo no; fi;)
-ifeq (x$(arm_toolchain_check), xyes)
-$(info Prepare arm toolchain ...);
-$(shell mkdir -p ../tools/toolchain/gcc-linaro-7.2.1-2017.11-x86_64_arm-linux-gnueabi || exit 1)
-$(shell tar --strip-components=1 -xf ../tools/toolchain/gcc-linaro-7.2.1-2017.11-x86_64_arm-linux-gnueabi.tar.xz -C ../tools/toolchain/gcc-linaro-7.2.1-2017.11-x86_64_arm-linux-gnueabi || exit 1)
-endif
-
-
-ifeq (x$(CONFIG_RISCV), xy)
-ifeq (x$(CONFIG_CPU_E90X), xy)
-CROSS_COMPILE=$(srctree)/../tools/toolchain/$(RISCV_PATH)/bin/riscv64-unknown-linux-gnu-
-DTS_PATH := $(PWD)/arch/riscv/dts
-else ifeq (x$(CONFIG_ARCH_RV32I), xy)
-CROSS_COMPILE := $(srctree)/../tools/toolchain/$(RISCV_PATH)/bin/riscv32-unknown-linux-
-DTS_PATH := $(PWD)/arch/riscv/dts
-else
-CROSS_COMPILE := $(srctree)/../tools/toolchain/$(RISCV_PATH)/bin/riscv64-unknown-linux-gnu-
-DTS_PATH := $(PWD)/arch/riscv/dts
-endif
-endif
-
-ifeq (x$(CONFIG_ARM), xy)
-CROSS_COMPILE := $(srctree)/../tools/toolchain/gcc-linaro-7.2.1-2017.11-x86_64_arm-linux-gnueabi/bin/arm-linux-gnueabi-
-DTS_PATH := $(PWD)/arch/arm/dts
-endif
-
-CROSS_COMPILE ?= $(srctree)/../tools/toolchain/gcc-linaro-7.2.1-2017.11-x86_64_arm-linux-gnueabi/bin/arm-linux-gnueabi-
-DTS_PATH ?= $(PWD)/arch/arm/dts
-
 #######################################################################
 # set default to nothing for native builds
 ifeq ($(HOSTARCH),$(ARCH))
-- 
2.47.1

